Spigot-API-Patches/0095-Add-method-to-open-already-placed-sign.patch

```diff
@@ -0,0 +1,28 @@
+From d084adbf8db14f500b342dd423605e7e0312a12a Mon Sep 17 00:00:00 2001
+From: Mark Vainomaa <mikroskeem@mikroskeem.eu>
+Date: Sun, 1 Apr 2018 02:28:43 +0300
+Subject: [PATCH] Add method to open already placed sign
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 87a9b750..7b6e71d4 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1728,6 +1728,14 @@ public interface Player extends HumanEntity, Conversable, CommandSender, Offline
+      */
+     void setPlayerProfile(PlayerProfile profile);
+     // Paper end
++    // Paper start - Add method to open already placed sign
++    /**
```

Comment:
Engage javadoc nitpick mode! (since this needs to be moved up to super anyway)

Opens an editor window for the specified sign  -- or similar if you can make my words sound better
@param sign **The** sign to open